notebooks/community/managed_notebooks/sentiment_analysis/Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,1044 @@
+{
```

Comment:
<p>There doesn't appear to be anything specific to Vertex AI Workbench in this notebook, and we prefer to have notebooks that work across multiple environments.</p><p> </p><p>Would you be able to check that it works with Colab (using the additional code for authentication provided in the notebook template <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb" target="_blank">here</a>)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>resolved</p>

Reply:
<p>we have added code to make sure it run on colab</p>